doc/base/classes.xml

```diff
@@ -5083,10 +5083,10 @@
 </class>
 <class name="Area2D" inherits="CollisionObject2D" category="Core">
 	<brief_description>
-		General purpose area detection and influence for 2D physics.
+	  General purpose area detection for 2D physics. Can override 2D physics properties within range.
```

Comment:
2 leading spaces to remove

Reply:
So the description should be on the same indent as the tag?

Reply:
No, it should be one more, and only tabs (2 spaces in your commit). You can see the original indent in red: 2 tabs